assets/js/components/legacy-notifications/dashboard-modules-alerts.js

```diff
@@ -74,7 +74,7 @@ class DashboardModulesAlerts extends Component {
 						title={ notification.title || '' }
 						description={ notification.description || '' }
 						blockData={ notification.blockData || [] }
-						winImage={ notification.winImage ? `${ global._googlesitekitLegacyData.admin.assetsRoot }images/${ notification.winImage }` : '' }
+						WinImageSVG={ notification.WinImageSVG || null }
```

Comment:
If `WinImageSVG` is false-y then it won't render anything, so it's safe to omit the `|| null` here as the `Notification` component will handle false-y `WinImageSVG` prop values for us 🙂 
```suggestion
						WinImageSVG={ notification.WinImageSVG }
```